README.md

```diff
@@ -1,2 +1,43 @@
-# KOBAW
-KOB Association Wallet
+
+| Title | KOB Association Wallet |
+|--- | ---|
+| Version | TBD |
+
+
+# Abstract   
+
+The KOB Association Wallet is an association/organisational wallet for the city of Kochi. This holds the verifiable credentials about registered entities (organisations) under a particular jurisdiction. And this is a Web Application hosted on the Community Cloud for the City of Kochi. A wallet provides an interface for storing and accessing identity information.
+
+# Dependent Projects
+
+*TBD*
+
+# Motivation
+
+The KOB Association wallet is a new service uses cutting-edge blockchain-based technology, which provides the foundation to help the organisations and associations in city of Kochi to be digitalized. This project is a part of Kochi OrgBook that is aimed to streamline and modernize services for Kochi.
+
+With this, users can quickly verify if an organization is legally registered to do business or provide services as a corporation. The motivation is to make verifable data such as the permits, licenses, and other accreditations that these organizations hold to be accesed through a decentralized way.
+
+# Status of the project
+
+Incubation
+
+# Solution
+
+The KOB Association Wallet is designed for enterprises to store the identity information in a decentralized way. The users can access the services from anywhere at anytime makes the wallet a user compatible facility.
+Every agent is paired with a digital wallet which is a key management system. This can be anything from a very simple static file on an embedded device to a highly sophisticated enterprise-grade key server. Regardless of the complexity, the job of a wallet is to safeguard sensitive data: key pairs, zero-knowledge proof blinded secrets, verifiable credentials, and any other cryptographic material needed to establish and maintain technical trust.
```

Comment:
Every agent is paired with a digital wallet which is a key management system. This can be anything from a very simple static file on an embedded device to a highly sophisticated enterprise-grade key server

I think this content is irrelevant here.